airflow/utils/sqlalchemy.py

```diff
@@ -153,6 +158,75 @@ def process_result_value(self, value, dialect):
         return BaseSerialization.deserialize(value)
 
 
+def sanitize_for_serialization(obj: V1Pod):
```

Comment:
this is copied from https://github.com/kubernetes-client/python/blob/master/kubernetes/client/api_client.py#L202

i'll point out the only differenc below (line 203)